stream2hop/Utils.py

```diff
@@ -32,3 +35,16 @@ def writeToScimma(payload, root, sc):
     global scimmaConfFile
     voevent = models.VOEvent.from_xml(payload)
     sc.write(voevent.asdict())
+
+
+def add_common_arguments(parser):
```

Comment:
May be worth calling this hop to keep it consistent throughout.

Reply:
Hmm, this was pointing at the right spot when I was doing the review. I think what I'm referring to is a few lines up.

Reply:
You mean the global variable's name, right?

Reply:
or all the scimma wordings in general?

Reply:
All the scimma wording in general if it makes sense. I realize also that some of them may still be scattered throughout and it may not be worth changing them and introducing a bug unnecessarily in this PR if they were already present.

Reply:
I changed them all to hop